### DISCUSSION TITLE
Sending action, collectionName, user details in webhook REST api POST request

### DISCUSSION THREAD

**Discussion by rakhi-sixt** [directus/directus#8523] [Archive]
Hi,
I am able to create a webhook and able to make REST api POST call on webhook action configured. I am getting request payload which contains collection updated content only.

Is there any way to send action(create/update/delete), collectionName (where action has been performed), user details (who performed action on collection) in request payload of configured POST API call in webhook?

Thanks,
Rakhi

**Comment by rijkvanzanten** [comment]
If you have the "send data" toggle enabled, the following fields will be sent to the receiving server:

https://github.com/directus/directus/blob/320a9cb1b4bdc5a48b250c9a1ca07d716275a239/api/src/webhooks.ts#L49-L55

**Reply by rakhi-sixt** [reply]
Thanks for your response @rijkvanzanten .
I am not able to see "send data" toggle on the create webhook dashboard. Is it a Directus version issue? My Directus version is **8.6.0**
<img width="1399" alt="Screenshot 2021-10-05 at 9 58 55 AM" src="https://user-images.githubusercontent.com/72124358/135960435-7d6e9029-7a84-4640-bb5e-e8cf59a7f7e5.png">


And as I said earlier also, I am able to receive only **update collection content in request payload of configured POST API call in webhook**

Let me know, if something else is also required to be done here.

**Reply by azrikahar** [reply]
@rakhi-sixt granted I'm personally not familiar with Directus v8 so I could be wrong, but this aforementioned "Send Event Data" might only be available in v9. Here's a sample screenshot of it:

![chrome_Xu4CyEruig](https://user-images.githubusercontent.com/42867097/135961832-be2a14df-9f56-4167-a241-c7d0eb6a1efb.png)

**Comment by joselcvarela** [comment]
Hey @rakhi-sixt just wanted to add that this repo is about v9 version.
You can find v8 repo is here https://github.com/directus/v8-archive 
Although is not being maintained.


If you want to upgrade to v9 you can follow this sections of the documentation https://docs.directus.io/guides/migrating-from-v8/#migrating-from-v8
Notice that the v9 version runs on Node, while v8 runs on PHP.
